Resync WPT for CSS pseudo class

rniwa opened the pull request:
#### b47ccd9be5cec5af0913161c2b642e7277e77d42
<pre>
Resync WPT for CSS pseudo class
<a href="https://bugs.webkit.org/show_bug.cgi?id=243677">https://bugs.webkit.org/show_bug.cgi?id=243677</a>

Reviewed by Tim Nguyen.

Updated web platform tests for CSS pseudo class as of 5c79743a93cfc4ded75d99338225531936da0c56.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-012-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-014-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-016-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-018-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-021-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-025-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-027-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-031-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-041-expected-mismatch.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/backdrop-animate-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/cascade-highlight-005-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/chrome-bug-1299142-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-bdi-element-expected.txt: Renamed
  from LayoutTests/fast/css/dir-pseudo-on-bdi-element-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-bdi-element.html: Renamed
  from LayoutTests/fast/css/dir-pseudo-on-bdi-element.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-input-element-expected.txt: Renamed
  from LayoutTests/fast/css/dir-pseudo-on-input-element-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-input-element.html: Renamed
  from LayoutTests/fast/css/dir-pseudo-on-input-element.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/first-line-green-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/first-line-nested-gcs-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/first-line-nested-gcs.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-002-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-006-expected.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-006-ref.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-006.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-inheritance-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-inheritance.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-visited-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-visited.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-currentcolor-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-currentcolor-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-currentcolor-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-soft-hyphens-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-soft-hyphens-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-soft-hyphens-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-004-expected-mismatch.html: Renamed
  from LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-004.tentative-expected-mismatch.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-004.html: Renamed
  from LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-004.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-005-expected.html: Renamed
  from LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-005.tentative-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-005.html: Renamed
  from LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-005.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-inheritance-computed-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-inheritance-computed-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-visited-computed-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-visited-computed-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/lang-pseudo-class-across-shadow-boundaries-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/lang-pseudo-class-across-shadow-boundaries-ref.html: Rename
  from LayoutTests/fast/shadow-dom/lang-pseudo-class-across-shadow-boundaries-expected.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/lang-pseudo-class-across-shadow-boundaries.html: Rename
  from LayoutTests/fast/shadow-dom/lang-pseudo-class-across-shadow-boundaries.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-hyphens.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-line-height-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-line-height-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-line-height.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-text-transform-dynamic-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-text-transform-dynamic-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-text-transform-dynamic.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/placeholder-as-multicol-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-intercharacter-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-intercharacter-012-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-overlay-and-grammar-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-overlay-and-spelling-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-universal-shadow-dom-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-universal-shadow-dom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-007-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-008-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-text-decoration-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-text-decoration-001-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-text-decoration-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/textpath-selection-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/textpath-selection-011-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/textpath-selection-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/w3c-import.log:
* LayoutTests/platform/win/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-input-element-expected.txt: Renamed
  from LayoutTests/platform/mac-wk1/fast/css/dir-pseudo-on-input-element-expected.txt.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/253318@main">https://commits.webkit.org/253318@main</a>
</pre>

nt1m reviewed (approved):
r=me if EWS is green

rniwa committed:
Resync WPT for CSS pseudo class
https://bugs.webkit.org/show_bug.cgi?id=243677

Reviewed by Tim Nguyen.

Updated web platform tests for CSS pseudo class as of 5c79743a93cfc4ded75d99338225531936da0c56.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-012-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-014-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-016-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-018-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-021-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-025-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-027-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-031-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/active-selection-041-expected-mismatch.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/backdrop-animate-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/cascade-highlight-005-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/chrome-bug-1299142-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-bdi-element-expected.txt: Renamed
  from LayoutTests/fast/css/dir-pseudo-on-bdi-element-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-bdi-element.html: Renamed
  from LayoutTests/fast/css/dir-pseudo-on-bdi-element.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-input-element-expected.txt: Renamed
  from LayoutTests/fast/css/dir-pseudo-on-input-element-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-input-element.html: Renamed
  from LayoutTests/fast/css/dir-pseudo-on-input-element.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/first-line-green-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/first-line-nested-gcs-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/first-line-nested-gcs.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-002-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-006-expected.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-006-ref.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-006.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-inheritance-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-inheritance.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-visited-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed-visited.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-painting-properties-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-explicit-default-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-currentcolor-root-implicit-default-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-currentcolor-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-currentcolor-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-currentcolor-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-soft-hyphens-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-soft-hyphens-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-painting-soft-hyphens-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-004-expected-mismatch.html: Renamed
  from LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-004.tentative-expected-mismatch.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-004.html: Renamed
  from LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-004.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-005-expected.html: Renamed
  from LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-005.tentative-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-005.html: Renamed
  from LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-paired-cascade-005.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-inheritance-computed-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-inheritance-computed-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-visited-computed-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-visited-computed-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/lang-pseudo-class-across-shadow-boundaries-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/lang-pseudo-class-across-shadow-boundaries-ref.html: Rename
  from LayoutTests/fast/shadow-dom/lang-pseudo-class-across-shadow-boundaries-expected.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/lang-pseudo-class-across-shadow-boundaries.html: Rename
  from LayoutTests/fast/shadow-dom/lang-pseudo-class-across-shadow-boundaries.html
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-hyphens.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-line-height-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-line-height-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-line-height.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-text-transform-dynamic-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-text-transform-dynamic-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/marker-text-transform-dynamic.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/placeholder-as-multicol-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-intercharacter-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-intercharacter-012-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-link-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-overlay-and-grammar-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-overlay-and-spelling-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-universal-shadow-dom-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/selection-universal-shadow-dom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-007-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-008-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-text-decoration-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-text-decoration-001-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/target-text-text-decoration-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/textpath-selection-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/textpath-selection-011-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/textpath-selection-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-pseudo/w3c-import.log:
* LayoutTests/platform/win/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/css/css-pseudo/dir-pseudo-on-input-element-expected.txt: Renamed
  from LayoutTests/platform/mac-wk1/fast/css/dir-pseudo-on-input-element-expected.txt.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/253318@main
